fe/fe-core/src/main/java/com/starrocks/scheduler/TaskRunManager.java

```diff
@@ -66,13 +66,13 @@ public SubmitResult submitTaskRun(TaskRun taskRun, ExecuteOption option) {
         for (Long taskId : pendingTaskRunMap.keySet()) {
             PriorityBlockingQueue<TaskRun> taskRuns = pendingTaskRunMap.get(taskId);
             if (taskRuns != null && !taskRuns.isEmpty()) {
-                validPendingCount++;
+                validPendingCount += taskRuns.size();
```

Comment:
what's the effect ?

Reply:
Original `Config.task_runs_queue_length`(default 500) only considers different task sizes, but I think it's more reasonable to consider different task sizes and their contained pending task runs.